### PR TITLE
fix(hosting): warn when `php` version is EOL

### PR DESCRIPTION
**Pull request by TheGoat-ma**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Ticket             | MANAGER-8769
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Customer is warn when php version is EOL, he should also be warn when he use a stable version

### :flags: Translations

- [x] TRANS-41727

### CODE REVIEW

**Review by JeremyDec — CHANGES_REQUESTED** [review]
Hello @TheGoat-ma, you don't have to change translation file [...]Messages_fr_CA.json, can you please remove this update please? It will be generated through translation process

**Review by JeremyDec — DISMISSED** [review]

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html [hunk 1]
```diff
@@ -368,14 +367,12 @@
                         data-term="{{:: 'hosting_dashboard_service_cdn' | translate }}"
                     >
                         <oui-tile-description>
-                            <span
-                                data-ng-if=""
-                            >
+                            <span data-ng-if="">
```
Comment (mohammed-zahaf) [inline packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html:370]: i know it was here before, but it's useless i think.

**Comment by antleblanc** [comment]
Pulling translations.

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html (hunk 1 above)
Comment (antleblanc) [reply packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html:370]: Thanks @mohammed-zahaf  for pointing out this useless element.
We'll handle it in a separate Pull Request.
Thanks!

**Comment by antleblanc** [comment]
Rebasing on `develop` in order to solve existing conflicts.
```txt
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_de_DE.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_en_GB.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_es_ES.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_it_IT.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_pl_PL.json
packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_pt_PT.json
```

**Comment by antleblanc** [comment]
Rebased successfully on `develop` branch. Conflicts are now solved.

**Review by antleblanc — DISMISSED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**